frontend/pages/queries/ManageQueriesPage/components/QueriesTable/QueriesTable.tests.tsx

```
@@ -224,12 +222,78 @@ describe("QueriesTable", () => {
+          currentUser: createMockUser(),
+          config: { partnerships: { enable_primo: true } },
+        },
+      },
+    });
+
+    render(
+      <QueriesTable
+        queries={[]}
+        totalQueriesCount={0}
+        hasNextResults={false}
+        curTeamScopeQueriesPresent={true}
```

ESLint

[error] Value must be omitted for boolean attributes

(react/jsx-boolean-value)